src/com/mqtt_paho/MQTTClient.cpp

```diff
@@ -188,9 +188,9 @@ int CMQTTClient::initClient() {
   mClientConnectionOptions.onFailure = onMqttConnectionFailed;
   mClientConnectionOptions.context = this;
 
+  std::string username;
```

Comment:
I have no idea why MSVC needs a change like this to make MQTT authentication work.

Reply:
this is really strange.